client/integrationTest/nodeTests/integrationNode.test.ts

```diff
@@ -203,7 +203,10 @@ describe('Integration Tests for Node Smart Contracts', () => {
                 await integrationTestUtil.generateSmartContractTests(smartContractName, '0.0.1', language, FabricRuntimeUtil.LOCAL_FABRIC);
                 // TODO: the generated smart contract tests for a generated smart contract can never pass out of the box
```

Comment:
can you update the comment as it now only applies to javascript

Reply:
Updated!